website/docs/docs/build/snapshots.md

```diff
@@ -364,13 +364,16 @@ snapshots:
       unique_key: order_id
       strategy: check
       check_cols:
-        - updated_at
+        - status
+        - is_cancelled
+      updated_at: updated_at
 ```
 
 In this example:
 
-- `check_cols: updated_at` makes sure that only the `updated_at` column triggers new snapshots.
+- If there is any change in at least one of the specified `check_cols`, then a new row is created in the snapshot. If the provided `updated_at` column value is not null, it will be used; otherwise, the timestamp will be used.
```

Comment:
```suggestion
- If at least one of the specified `check_cols `changes, the snapshot creates a new row. If the `updated_at` column has a value (is not null), the snapshot uses it; otherwise, it defaults to the timestamp.
```